gdpr/vendorlist-fetching.go

```diff
@@ -11,35 +11,37 @@ import (
 	"time"
 
 	"github.com/golang/glog"
+	"github.com/prebid/go-gdpr/api"
 	"github.com/prebid/go-gdpr/vendorlist"
+	"github.com/prebid/go-gdpr/vendorlist2"
 	"github.com/prebid/prebid-server/config"
 	"golang.org/x/net/context/ctxhttp"
 )
 
-type saveVendors func(uint16, vendorlist.VendorList)
+type saveVendors func(uint16, api.VendorList)
 
 // This file provides the vendorlist-fetching function for Prebid Server.
 //
 // For more info, see https://github.com/prebid/prebid-server/issues/504
 //
 // Nothing in this file is exported. Public APIs can be found in gdpr.go
 
-func newVendorListFetcher(initCtx context.Context, cfg config.GDPR, client *http.Client, urlMaker func(uint16) string) func(ctx context.Context, id uint16) (vendorlist.VendorList, error) {
+func newVendorListFetcher(initCtx context.Context, cfg config.GDPR, client *http.Client, urlMaker func(uint16, int) string, tCFVer int) func(ctx context.Context, id uint16) (vendorlist.VendorList, error) {
 	// These save and load functions can be used to store & retrieve lists from our cache.
 	save, load := newVendorListCache()
 
 	withTimeout, cancel := context.WithTimeout(initCtx, cfg.Timeouts.InitTimeout())
 	defer cancel()
 	populateCache(withTimeout, client, urlMaker, save)
 
-	saveOneSometimes := newOccasionalSaver(cfg.Timeouts.ActiveTimeout())
+	saveOneSometimes := newOccasionalSaver(cfg.Timeouts.ActiveTimeout(), ctfVer)
```

Comment:
Is ctfVer a typo of tcfVer? Those should be easy to sort through once the go-gdpr library is updated. I'll comment no further on it.

Reply:
Yes, I am surprised the parser did not flag this.